conf/messages

```diff
@@ -1660,18 +1918,32 @@ rebaseAcquisitionPrice.error.tooLarge=The property’s market value must be an a
 rebaseAcquisitionPrice.error.tooManyDecimals=The property’s market value must be an amount of money with pence as 2 decimal places or less, like 160,000 or 23,040.50
 rebaseAcquisitionPrice.dropdown.header=Why you need to enter this amount
 
+rebaseAcquisitionPrice.indirect.error.required=Enter the market value of the shares on 5 April 2019
```

Comment:
if you do a CTRL+F on `April` you should find all the instances where you've still got hard coded dates